### PR TITLE
Enables UISegmentedControl touch

### PR DESCRIPTION
**Pull request by mumme**
@rastersize

### CODE REVIEW

File: sources/HUBComponentWrapper.m [hunk 1]
```diff
@@ -432,6 +432,11 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceive
     UIView *currentView = touch.view;
     
     while (currentView != nil && currentView != self.view) {
+
```
Comment (rastersize) [inline sources/HUBComponentWrapper.m]: 🎨 Remove the empty line

File: sources/HUBComponentWrapper.m [hunk 2]
```diff
@@ -432,6 +432,11 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceive
...
+        if ([currentView isKindOfClass:[UISegmentedControl class]]) {
```
Comment (rastersize) [inline sources/HUBComponentWrapper.m]: Wondering if we should not add this check and instead change the `UIButton` check to `UIControl`?

@mhallendal @cerihughes What do you and CA think?

**Comment by marmelroy** [comment]
I was intrigued by the fact that we were traversing the view hierarchy looking for specific subclasses and created an alternative PR - https://github.com/spotify/HubFramework/pull/285

**Comment by rastersize** [comment]
I’d be in favour of merging this (after changing to look for `UIControl`) and the concept of #285 into a single change. That is:
1. Look for a gesture controller ala. #285 if not found proceed to,
2. Go through the view hierarchy and look for a `UIControl`.

**Comment by rastersize** [comment]
@mhallendal @marmelroy @mumme What do you think of this ☝️

**Pull request merged**